tests/conftest.py

```diff
@@ -130,6 +130,34 @@ def forward(self, signal: torch.Tensor, spectrogram: torch.Tensor):
 )
 
 
+# create model with dynamic input / output node
+
+class TorchModelDynamic(torch.nn.Module):
+
+    def __init__(
+        self,
+    ):
+        super().__init__()
+
+    def forward(self, x: torch.Tensor):
+        return x
+
+
+pytest.MODEL_DYNAMIC_PATH = os.path.join(pytest.TMP, 'dynamic.onnx')
+torch.onnx.export(
+    TorchModelDynamic(),
+    torch.randn(pytest.FEATURE_SHAPE),
+    pytest.MODEL_DYNAMIC_PATH,
+    input_names=['input'],
+    output_names=['output'],
+    dynamic_axes={
+        'input': {1: 'time'},
+        'output': {1: 'time'},
+    },
```

Comment:
What happens if I export using

```python
    dynamic_axes={
        'input': {1: 'magnitude'},
        'output': {1: 'magnitude'},
    },
```

Reply:
Changed it accordingly and updated the description.